build/AzurePipelinesTemplates/WindowsAppSDK-BuildProject-Steps.yml

```diff
@@ -99,6 +99,20 @@ steps:
       arguments: -CheckTAEFService -Offline -Verbose
       workingDirectory: '$(Build.SourcesDirectory)'
 
+  - task: powershell@2
+    name: UpdateTraceloggingConfig
+    inputs:
+      targetType: 'inline'
+      script: |
+        $srcPath = get-childitem -path 'dev\WindowsAppSDK_Insights\packages' -File 'MicrosoftTelemetry.h' -Recurse
```

Comment:
NIT: Case

```
...Get-ChildItem -Path...
```

Yes Powershell is case-insensitive but harder to follow where you mix lowercase and MixedCase in the same command. We typically use MixedCase for our Powershell so...